lib/src/components/files/fileCard/FileCard.js

```diff
@@ -1,44 +1,50 @@
 import React from 'react';
 import { bool, node } from 'prop-types';
 import cx from 'classnames';
-import Card from 'lib/Card';
-
-function FileCard({ preview, meta, insetMeta, children, ...cardProps }) {
-  const cardClassNames = `file-card group ${cardProps.className}`;
-
-  const metaContainerClassNames = cx(
-    'border border-neutral-90 border-solid border-b-0 border-l-0 border-r-0',
-    {
-      'absolute bottom-0 w-full bg-white -mb-40 group-hover:mb-0 transition-mb duration-200': insetMeta
-    }
-  );
+import { Card } from 'lib/src/components/card/Card';
+
+function FileCard({
+  preview,
+  meta,
+  controls,
+  insetMeta,
+  children,
+  size,
+  ...cardProps
+}) {
+  const cardClassNames = cx('file-card group', cardProps.className, {
+    'w-full': size !== 'thumbnail',
+    'w-8 h-8': size === 'thumbnail',
+    'file-card-inset-meta': insetMeta
+  });
 
   return (
     <Card
       {...cardProps}
       className={cardClassNames}
-      innerClassNames="relative shadow-small overflow-hidden rounded"
+      innerClassNames="relative overflow-hidden rounded"
     >
-      <Card.Content collapse>{preview}</Card.Content>
+      <Card.Content className="p-0">{preview}</Card.Content>
 
-      {meta && (
-        <Card.Content className={metaContainerClassNames}>{meta}</Card.Content>
-      )}
+      {meta && <Card.Content className="file-card-meta">{meta}</Card.Content>}
 
+      {controls && <Card.Controls>{controls}</Card.Controls>}
```

Comment:
Some recent feedback from @altaudio was making it clearer about where "controls" live in the card / filecard namespace. This makes it easier to understand.